src/main/java/com/google/cloud/tools/ide/login/JavaPreferenceOAuthDataStore.java

```diff
@@ -0,0 +1,106 @@
+/*******************************************************************************
+ * Copyright 2014 Google Inc. All Rights Reserved.
+ * <p>
+ * All rights reserved. This program and the accompanying materials
```

Comment:
Ah, this is weird. All the other files have this EPL license.

Also, I forgot the change the year too.